### PR TITLE
Fixed some typos: BOX64 -> BOX86

### PR DESCRIPTION
**Pull request by MikeWang000000**

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -903,8 +903,8 @@ void PrintFlags() {
     printf(" BOX86_LIBGL=libXXXX set the name (and optionnaly full path) for libGL.so.1\n");
     printf(" BOX86_LD_PRELOAD=XXXX[:YYYYY] force loading XXXX (and YYYY...) libraries with the binary\n");
     printf(" BOX86_ALLOWMISSINGLIBS with 1 to allow to continue even if a lib is missing (unadvised, will probably crash later)\n");
-    printf(" BOX64_PREFER_EMULATED=1 to prefer emulated libs first (execpt for glibc, alsa, pulse, GL, vulkan and X11)\n");
-    printf(" BOX64_PREFER_WRAPPED if box86 will use wrapped libs even if the lib is specified with absolute path\n");
+    printf(" BOX86_PREFER_EMULATED=1 to prefer emulated libs first (execpt for glibc, alsa, pulse, GL, vulkan and X11)\n");
```
Comment (MikeWang000000) [inline src/main.c:906]: Ref:
https://github.com/ptitSeb/box86/blob/30c72e23bd3f11f7081ff8bea95103d762020001/src/main.c#L1027

File: src/main.c [hunk 2]
```diff
@@ -1321,7 +1321,7 @@ int main(int argc, const char **argv, char **env)
 
     // check BOX86_LOG debug level
     LoadLogEnv();
-    if(!getenv("BOX64_NORCFILES")) {
+    if(!getenv("BOX86_NORCFILES")) {
```
Comment (MikeWang000000) [inline src/main.c:1324]: Ref:
https://github.com/ptitSeb/box86/blob/30c72e23bd3f11f7081ff8bea95103d762020001/docs/USAGE.md?plain=1#L329

File: src/main.c [hunk 3]
```diff
@@ -903,8 +903,8 @@ void PrintFlags() {
     printf(" BOX86_LIBGL=libXXXX set the name (and optionnaly full path) for libGL.so.1\n");
     printf(" BOX86_LD_PRELOAD=XXXX[:YYYYY] force loading XXXX (and YYYY...) libraries with the binary\n");
     printf(" BOX86_ALLOWMISSINGLIBS with 1 to allow to continue even if a lib is missing (unadvised, will probably crash later)\n");
-    printf(" BOX64_PREFER_EMULATED=1 to prefer emulated libs first (execpt for glibc, alsa, pulse, GL, vulkan and X11)\n");
-    printf(" BOX64_PREFER_WRAPPED if box86 will use wrapped libs even if the lib is specified with absolute path\n");
+    printf(" BOX86_PREFER_EMULATED=1 to prefer emulated libs first (execpt for glibc, alsa, pulse, GL, vulkan and X11)\n");
+    printf(" BOX86_PREFER_WRAPPED if box86 will use wrapped libs even if the lib is specified with absolute path\n");
```
Comment (MikeWang000000) [inline src/main.c:907]: Ref:
https://github.com/ptitSeb/box86/blob/30c72e23bd3f11f7081ff8bea95103d762020001/src/main.c#L1026

**Comment by MikeWang000000** [comment]
Some environment variables like `BOX86_NORCFILES` did not work because of typos. From now it will work.

**Comment by ptitSeb** [comment]
nice, thnaks.

**Pull request merged**